contracts/AnchoredView/AnchoredView.sol

```diff
@@ -0,0 +1,278 @@
+pragma solidity ^0.6.6;
+pragma experimental ABIEncoderV2;
+
+import "./SymbolConfiguration.sol";
+import "../OpenOraclePriceData.sol";
+
+interface CErc20 {
+    function underlying() external view returns (address);
+}
+
+interface AnchorOracle {
```

Comment:
is there an example of an anchor that will be used ?

Reply:
great question. 
 https://etherscan.io/address/0x02557a5e05defeffd4cae6d83ea3d173b272c904#code
https://github.com/compound-finance/compound-oracle/blob/master/contracts/PriceOracle.sol